Fix OpenTelemetry k8sevents RBAC (#111543)
openshift/openshift-docs: modules/otel-receivers-kubernetesevents-receiver.adoc

[role="_abstract"]
The Kubernetes Events Receiver collects events from the Kubernetes API server. The collected events are converted into logs.

> The following {ocp-product-title} permissions are required for the Kubernetes Events Receiver. Alternatively, a namespace-scoped `Role` and `RoleBinding` can be used when the receiver is configured to watch specific namespaces via the `namespaces` option.

Reviewer: ```suggestion
The following {ocp-product-title} permissions are required for the Kubernetes Events Receiver. Alternatively, you can use a namespace-scoped `Role` object and `RoleBinding` object when you configure the receiver to watch specific namespaces by using the `namespaces` option.
```
